### PR TITLE
Add support to set maxBitRateKbps for local video and content share

### PR DESCRIPTION
**Pull request by ziyiz-amzn**
## ℹ️ Description
Provide a way to configure video send encoding parameters when start local video and content share.

### Issue #, if available
Chime-48548

### Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update
    - [x] README update
    - [x] CHANGELOG update
    - [x] guildes update
- [ ] This change requires a dependency update
    - [x] Amazon Chime SDK Media
    - [ ] Other (update corresonding legal documents)

## 🧪 How Has This Been Tested?
Start local video with different maxBitRateKbps values, verified the metrics `videoSendBitrate` is closed to the maxBitRateKbps value.

### Additional Manual Test
- [x] Pause and resume remote video
- [x] Switch local camera
- [x] Rotate screen back and forth

## 📱 Screenshots, if available

Image from iOS (1)
Image from iOS (3)
Image from iOS (2)

#### Integration validation (required before release)

- [x] Unit tests pass
- [ ] Build SDK against simulator with release options
- [ ] Build SDK against device with release options
- [x] Build SDK against simulator with debug options
- [ ] Build SDK against device with debug options
- [ ] Test Demo against simulator with SDK (debug build) in workspace
- [x] Test Demo against device with SDK (debug build) in workspace
- [ ] Test DemoObjC against simulator with SDK (debug build) in workspace
- [ ] Test DemoObjC against device with SDK (debug build) in workspace
- [ ] Test Demo against simulator with SDK.framework (release build)
- [ ] Test Demo against device with SDK.framework (release build)
- [ ] Test DemoObjC against simulator with SDK.framework (release build)
- [ ] Test DemoObjC against device with SDK.framework (release build)

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: AmazonChimeSDK/AmazonChimeSDK/audiovideo/video/LocalVideoConfiguration.swift [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+import Foundation
+
+/// Configuration for a local video or content share to be sent
+@objcMembers public class LocalVideoConfiguration: NSObject {
+
+    /// The max bit rate for video encoding. If null, video client will default it to 1400 kbps
+    /// and will scale down by numbers of video senders. Actual quality achieved may vary throughout the call
+    /// depending on what system and network can provide.
+    /// maxBitRate has to be greater than 0
+    public var maxBitRateKbps: Int32? {
```
Comment (hensmi-amazon) [inline AmazonChimeSDK/AmazonChimeSDK/audiovideo/video/LocalVideoConfiguration.swift:18]: If you want to be an abstraction pro, you could make a `DataRate` class with functions like `kbps()`, etc.

File: AmazonChimeSDK/AmazonChimeSDK/audiovideo/video/LocalVideoConfiguration.swift [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+        didSet {
+            if let bitRate = maxBitRateKbps, bitRate <= 0 {
```
Comment (georgezy-amzn) [inline AmazonChimeSDK/AmazonChimeSDK/audiovideo/video/LocalVideoConfiguration.swift:20]: Can we use `UInt32` so we don't need to worry about `bitRate <= 0` ?

**Review by linsang21 — COMMENTED** [review]
Pretty much same comments as on Android PR, let's make sure if changes should apply on both.

File: AmazonChimeSDKDemo/AmazonChimeSDKDemo/DeviceSelectionViewController.swift [hunk 3]
```diff
@@ -32,6 +32,8 @@ class DeviceSelectionViewController: UIViewController {
         videoPreviewImageView.mirror = model?.shouldMirrorPreview ?? false
         model?.cameraCaptureSource.addVideoSink(sink: videoPreviewImageView)
         model?.cameraCaptureSource.start()
+
+        setupHideKeyboardOnTap()
```
Comment (ziyiz-amzn) [inline AmazonChimeSDKDemo/AmazonChimeSDKDemo/DeviceSelectionViewController.swift:36]: Will delete this.

File: AmazonChimeSDK/AmazonChimeSDK/audiovideo/video/LocalVideoConfiguration.swift [hunk 4]
```diff
@@ -12,6 +12,16 @@ import Foundation
 @objcMembers public class LocalVideoConfiguration: NSObject {
 
     /// The flag to disable/enable simulcast, default to true
-    /// only for localVideo, not work for content share
-    public var simulcastEnabled: Bool = true
+    /// For local video use only, will not work for content share
+    public var simulcastEnabled: Bool
```
Comment (hokyungh) [inline AmazonChimeSDK/AmazonChimeSDK/audiovideo/video/LocalVideoConfiguration.swift:16]: why is this removed?

**Review by georgezy-amzn — APPROVED** [review]

File: AmazonChimeSDK/AmazonChimeSDK/audiovideo/video/LocalVideoConfiguration.swift (hunk 4 above)
Comment (ziyiz-amzn) [reply AmazonChimeSDK/AmazonChimeSDK/audiovideo/video/LocalVideoConfiguration.swift:16]: It is refactored into the constructor.

Comment (hokyungh) [reply AmazonChimeSDK/AmazonChimeSDK/audiovideo/video/LocalVideoConfiguration.swift:16]: ok. Just to note, it won't work on objective-C since it doesn't support default parameters.

**Review by hokyungh — APPROVED** [review]

File: AmazonChimeSDK/AmazonChimeSDK/audiovideo/video/LocalVideoConfiguration.swift (hunk 4 above)
Comment (ziyiz-amzn) [reply AmazonChimeSDK/AmazonChimeSDK/audiovideo/video/LocalVideoConfiguration.swift:16]: discussed offline. will merge.
